runtime/doc/msgpack_rpc.txt

```diff
@@ -404,4 +404,91 @@ of update.
 	The popupmenu is hidden.
 
 ==============================================================================
+7. Watching buffers						*rpc--watching-buffers*
+
+Special functions have been created to allow co-processes to be notified in
+real-time when a buffer's content changes. (It is difficult and innefficient
+to do this using |TextChanged| and other autocommands.)
+
+		*Todo	what about when a file is changed from outside vim?
+				(See |FileChangedShell|, |ShellCmdPost| and |ShellFilterPost|)
+
+Setting Up~
+
+Use |rpcstart()| to start up your co-process. This will give you a valid
```

Comment:
no need to repeat this here. It is likely also confusing: I suppose many consumers would be plugins living in a rplugin host, and don't be separate co-processes. (As the notifications are high-throughput we should also add multiplexing logic to e.g. the python-client, so that multiple plugins active in the same buffer can share the same notification stream)